main.py

```diff
@@ -0,0 +1,16 @@
+import hydra

```

Comment:
what is this file?

Reply:
was using for testing the new generated core lib, removed it now